encryption/caesar.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include "caesar.h"
+
+int euclideanModulo(int a, int b) {
```

Comment:
Let's be familiar with approach to not call function arguments in math style. People read code, change it and arguments give a lot of information about details to avoid some unclear changes.